events/responses.py

```diff
@@ -23,4 +23,16 @@
 
 events_not_found_404 = {
     "message": "Events couldn't be fetched."
+}
+
+event_registration_200 = {
+    "message":"Registration Successful"
+}
+
+event_does_not_exist_404 = {
+  "message": "Registration Failed. Event does not exist."
+}
+
+user_unauthorized_401 = {
+  "message":"Unauthorized User"
```

Comment:
```suggestion
  "message":"Please follow OTP verification before registering"
```
It is good to mention 'why?' when we can in the, response message.

Reply:
done